src/AtelierTomato.Markov.Service.Discord/DiscordObjectOIDBuilder.cs

```diff
@@ -5,15 +5,25 @@ namespace AtelierTomato.Markov.Service.Discord
 {
 	public class DiscordObjectOIDBuilder
 	{
-		public static async Task<DiscordObjectOID> Build(IGuild? guild, IChannel channel, ulong messageID, string instance = "discord.com")
+		/// <summary>
+		/// Builds a DiscordObjectOID at scope of <see cref="DiscordObjectOID.Channel"/> or <see cref="DiscordObjectOID.Thread"/> <paramref name="guild"/> and <paramref name="channel"/> are passed.
+		/// If <paramref name="messageID"/> is passed, builds at scope of <see cref="DiscordObjectOID.Message"/>.

```

Comment:
please warn (at the start of the comment imo) that this requires being connected to discord despite being a build method, it is a bit of a surprise